Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -49,6 +49,7 @@ public IEnumerable<ContentItemTemplate> Templates
         }
 
         public bool LaunchDebugger { get; set; }
+        public bool MultithreadMode { get; set; }
```

Comment:
Yea, just use the variable in `Settings.Instance` instead, `Settings` is a class that gets serialized and saved during the application exit so you don't need to do anything special to save a variable in there.